components/create_swagger_config.py

```diff
@@ -6,428 +6,434 @@
 """
 import argparse
-import json
 import getpass
+import json
 import os
 import re
+import sys
+import traceback
+
 import requests
 from requests.auth import HTTPBasicAuth
-import traceback
-import sys
 
 requests.packages.urllib3.disable_warnings()
 
-k_swaggerParamIsiPropCommonFields = [
-    "description", "required", "type", "default", "maximum", "minimum", "enum",
-    "items"]
+SWAGGER_PARAM_ISI_PROP_COMMON_FIELDS = [
+    'description', 'required', 'type', 'default', 'maximum', 'minimum', 'enum',
+    'items']
 
 # list of url parameters that need to be url encoded, this hack works for now,
 # but could cause problems if new params are added that are not unique.
-k_urlEncodeParams = ["NfsAliaseId"]
+URL_ENCODE_PARAMS = ['NfsAliaseId']
 # our extension to swagger which is used to generate code for doing the url
 # encoding of the parameters specified above.
-X_ISI_URL_ENCODE_PATH_PARAM = "x-isi-url-encode-path-param"
+X_ISI_URL_ENCODE_PATH_PARAM = 'x-isi-url-encode-path-param'
 
-g_operations = {} # tracks swagger ops generated from URLs to ensure uniquenes
+# tracks swagger operations generated from URLs to ensure uniqueness
+GENERATED_OPS = {}
 
 
-def OneFsShortVers(host, port, auth):
+def onefs_short_version(host, port, auth):
     """Query a cluster and return the 2 major version digits"""
-    url = "https://{0}:{1}/platform/1/cluster/config".format(host, port)
+    url = 'https://{0}:{1}/platform/1/cluster/config'.format(host, port)
     config = requests.get(url, auth=auth, verify=False).json()
-    release = config["onefs_version"]["release"].strip("v")
-    short_vers = ".".join(release.split(".")[:2])
+    release = config['onefs_version']['release'].strip('v')
+    short_vers = '.'.join(release.split('.')[:2])
     return short_vers
 
 
-def IsiPropsToSwaggerParams(isiProps, paramType):
-    if len(isiProps) == 0:
+def isi_props_to_swagger_params(isi_props, param_type):
+    """Convert isi properties to Swagger parameters."""
+    if not isi_props:
         return []
-    swaggerParameters = []
-    for isiPropName in isiProps:
+    swagger_parameters = []
+    for isi_prop_name in isi_props:
         # build a swagger param for each isi property
-        swaggerParam = {}
-        swaggerParam["in"] = paramType
-        swaggerParam["name"] = isiPropName
-        isiProp = isiProps[isiPropName]
+        swagger_param = {}
+        swagger_param['in'] = param_type
+        swagger_param['name'] = isi_prop_name
+        isi_prop = isi_props[isi_prop_name]
         # attach common fields
-        for fieldName in isiProp:
-            if fieldName not in k_swaggerParamIsiPropCommonFields:
-                print >> sys.stderr, "WARNING: " + fieldName + " not " \
-                        "defined for Swagger in prop: " + str(isiProp)
+        for field_name in isi_prop:
+            if field_name not in SWAGGER_PARAM_ISI_PROP_COMMON_FIELDS:
+                print('WARNING: {} not defined for Swagger in prop: {}'.format(
```

Comment:
Since we're using `print` as a function now, should we add `from __future__ import print_function` ?

Reply:
Yes, that's a good call. This will ensure consistency between Python 2 and Python 3.